app/controllers/claims/schools/claims_controller.rb

```diff
@@ -1,3 +1,55 @@
 class Claims::Schools::ClaimsController < ApplicationController
   include Claims::BelongsToSchool
+
+  def index
+    render locals: { claims: @school.claim_applications }
+  end
+
+  def new
+    claim = Claim.create!(school_id: params.require(:school_id), draft: true)
+    redirect_to claims_school_claim_path(id: claim.id)
+  end
+
+  def show
+    render locals: { claim: }
+  end
+
+  def edit
+    render locals: {
+      claim:,
+      mentor_trainings:,
+      step:,
+    }
+  end
+
+  def update
+    claim_form = ClaimForm.new(claim:, step:, claim_params:)
+
+    if claim_form.save
+      redirect_to claims_school_claim_path(id: claim.id)
+      flash[:success] = t(".claim_updated")
+    else
+      render :edit, locals: { claim:, mentor_trainings:, step: }
+    end
+  end
+
+  private
+
+  def mentor_trainings
+    claim.mentor_trainings.build if claim.mentor_trainings.blank?
+  end
+
+  def claim_params
+    params.require(:claim).permit(
+      mentor_trainings_attributes: %w[provider_id teacher_id id],
+    )
+  end
+
+  def claim
+    @claim ||= Claim.find(params.require(:id)).decorate
```

Comment:
Should we also scope this "find" to the current school?

```suggestion
    @claim ||= @school.claims.find(params.require(:id)).decorate
```